components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/AbstractAuthenticatorAdapter.java

```diff
@@ -119,6 +126,11 @@ public String getName() {
         return authenticatorName;
     }
 
+    public AuthenticatorPropertyConstants.AuthenticationType getAuthenticationType() {
+
+        return authenticationType;
+    }
```

Comment:
How this type is used ?

Reply:
This is used to check whether authenticator is IDENTIFICATION or VERIFICATION in process of userData from the action invocation response.